.github/workflows/update-version.yml

```diff
@@ -0,0 +1,46 @@
+name: Update library version
+
+on:
+  workflow_dispatch:
+    inputs:
+      new_version:
+        description: 'The new version (vX.Y.Z)'
+        required: true
+      target_branch:
+        description: 'Target branch for the version update'
+        required: true
+        type: choice
```

Comment:
Consider adding `v1.x` as a target branch option if the Symfony bundle maintains multiple branches like the PHP SDK does. If only `main` is used, this is perfect as-is.